contracts/interfaces/ISwapRouter.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+pragma abicoder v2;
+
+import './IUniswapV3SwapCallback.sol';
+
+
+
+/// @title Router token swapping functionality
+/// @notice Functions for swapping tokens via Uniswap V3
+interface ISwapRouter is IUniswapV3SwapCallback {
```

Comment:
are we using all the functions of this interface

Reply:
no, we are just using one function from this interface